parseAndPopulate/runCapabilities.py

```diff
@@ -184,25 +191,26 @@ def main(scriptConf=None):
         for pattern in patterns:
             for filename in find_files(args.dir, pattern):
                 update = True
+                # NOTE: Hash of file will be saved for each file individually, not only capability xml files
                 if not args.api and args.save_modification_date:
+                    files_modifications = {}
+                    path = '{}/filesModifications.json'.format(temp_dir)
+                    file_hash = hash_file(filename)
                     try:
-                        file_modification = open(temp_dir + '/fileModificationDate/' + '-'.join(filename.split('/')[-4:]) +
-                                                 '.txt', 'r')
-                        time_in_file = file_modification.readline()
-                        if time_in_file in str(time.ctime(os.path.getmtime(filename))):
+                        with open(path, 'r') as f:
+                            files_modifications = json.load(f)
+                        file_hash_old = files_modifications[filename]
+                        if file_hash_old == file_hash:
                             update = False
                             LOGGER.info('{} is not modified. Skipping this file'.format(filename))
```

Comment:
don t use the whole "update" variable. continue should be sufficient here after the logged information

Reply:
'Update' variable removed completely, replaced with continue statement.